connection/pom.xml

```diff
@@ -76,8 +76,8 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
```

Comment:
Do we need to set source and target here, would we not inherit this setting from the parent pom.xml? If we do we should not set source and target in any pom-file except the parent pom. If not we should set it in all the pom-files, or?

Reply:
If we set the source and target configuration in the parent POM, the child modules will inherit these values by default. So yeah, in this case we do not need to repeat the configuration in the individual child module POMs unless if we want to override the configuration for a specific module, as we were doing previously with Java 8, so maybe we can remove all of this configuration in the child modules.

https://maven.apache.org/pom.html#POM_Relationships
https://maven.apache.org/pom.html#Inheritance

Reply:
Then I propose that we remove it from all the child modules and just set it in the parent pom.